astropy/time/core.py

```diff
@@ -34,13 +34,6 @@
 __all__ = ['Time', 'TimeDelta', 'TIME_SCALES', 'TIME_DELTA_SCALES',
            'ScaleValueError', 'OperandTypeError']
 
-try:
-    # Not guaranteed available at setup time
-    from . import erfa_time
-except ImportError:
-    if not _ASTROPY_SETUP_:
-        raise
-
```

Comment:
This is an ignorant question, but are we guaranteed that `erfa` is available at setup time?